packages/kbn-handlebars/scripts/print_ast.js

```diff
@@ -5,20 +5,31 @@
  */
 'use strict'; // eslint-disable-line strict
 
+const { relative } = require('path');
 const { inspect } = require('util');
 
 const { parse } = require('handlebars');
+const argv = require('minimist')(process.argv.slice(2));
 
-const template = process.argv[2];
-const filter = (process.argv[3] || 'loc').split(',');
-const containsSubNodes = ['body', 'path', 'program', 'params'];
+const DEFAULT_FILTER = 'loc,strip,openStrip,inverseStrip,closeStrip';
+
+const filter = argv['show-all'] ? [''] : (argv.filter || DEFAULT_FILTER).split(',');
+const hideEmpty = argv['hide-empty'] || false;
+const template = argv._[0];
 
 if (template === undefined) {
-  console.log(`Usage: ${process.argv[1]} <template> <filter>`);
+  const script = relative(process.cwd(), process.argv[1]);
+  console.log(`Usage: ${script} [options] <template>`);
+  console.log();
+  console.log('Options:');
+  console.log('  --filter=...  A comma separated list of keys to filter from the output.');
+  console.log(`                Default: ${DEFAULT_FILTER}`);
+  console.log('  --hide-empty  Do not display empty properties.');
+  console.log('  --show-all    Do not filter out any properties. Equivalent to --filter="".');
+  console.log();
+  console.log('Example:');
+  console.log(`  ${script} --hide-empty -- 'hello {{name}}'`);
```

Comment:
My bad...I didn't see that auto-merge was on. This just being a nit, maybe we can just roll it into your next handlebars PR? Sorry about that.

Reply:
Hmm this might be shell specific, but in my shell (zsh) the `--` is required in order for the script to properly understand that the template isn't the value of `--hide-empty` but instead its own argument

Reply:
Yeah, never mind. I see what I did. The example is fine.